web/src/components/Plugin/UI/limit-count.tsx

```diff
@@ -323,6 +323,35 @@ const LimitCount: React.FC<Props> = ({ form, schema }) => {
           ))}
         </Select>
       </Form.Item>
+      <Form.Item
+        label="group"
+        name="group"
+        tooltip={formatMessage({ id: 'component.pluginForm.limit-count.key.tooltip' })}
+      >
+        <Input />
+      </Form.Item>
+      <Form.Item
+        initialValue={properties.allow_degradation.default}
+        label="allow_degradation"
+        name="allow_degradation"
+        tooltip={formatMessage({
+          id: 'component.pluginForm.limit-count.allow_degradation.tooltip',
+        })}
+        valuePropName="checked"
+      >
+        <Switch defaultChecked={properties.allow_degradation.default} />
+      </Form.Item>
```

Comment:
why `initialValue` and `defaultChecked` both exist?

Reply:
> why `initialValue` and `defaultChecked` both exist?

Sorry i forgot to delete the extra one.